### PR TITLE
chore(updatecli): allow separated `weekly` and `daily` cron frequency, with `launchable` as candidate

### PR DESCRIPTION
**Pull request by lemeurherve**
Extracted from #85

### CODE REVIEW

**Review by dduportal — APPROVED** [review]

**Review by dduportal — COMMENTED** [review]
weird, the CI says `File ./updatecli/updatecli.d: lstat ./updatecli/updatecli.d: no such file or directory` 🤔

**Comment by dduportal** [comment]
> > weird, the CI says `File ./updatecli/updatecli.d: lstat ./updatecli/updatecli.d: no such file or directory` 🤔
> 
> Strange indeed. When running updatecli locally I don't have this error.
> 
> ```
> Run Summary
> ===========
> Pipeline(s) run:
>   * Changed:    0
>   * Failed:     0
>   * Skipped:    1
>   * Succeeded:  7
>   * Total:      8
> ```

Then it means it's something wrong with the pipeline library for updatecli.

**Comment by lemeurherve** [comment]
Alas activating updatecli debug output didn't gave us more info.

**Comment by lemeurherve** [comment]
When restoring previous folder name:
image

**Comment by lemeurherve** [comment]
Oh... The default `config` value of the pipeline library function `updatecli` is './updatecli/updatecli.d', that's why it was failing.

File: Jenkinsfile_k8s [hunk 1]
```diff
@@ -41,7 +41,8 @@ parallel(
   'updatecli': {
     updatecli(action: 'diff')
```
Comment (dduportal) [inline Jenkinsfile_k8s:42]:
```suggestion
    updatecli(action: 'diff', config: './updatecli/updatecli.d/daily.d')
    updatecli(action: 'diff', config: './updatecli/updatecli.d/weekly.d')
```

Comment (lemeurherve) [reply Jenkinsfile_k8s:42]: Not needed anymore since I've kept udpatecli.d as root folder, available as default config :)

Comment (lemeurherve) [reply Jenkinsfile_k8s:43]: Wrote those comments at the same time: [`74825b6` (#87)](https://github.com/jenkins-infra/docker-inbound-agents/pull/87/commits/74825b6dea6de5c15470d048b5696188d5d47dc9), good to you?

**Review by dduportal — APPROVED** [review]
LGTM, don't forget to check the "apply" execution once merged.

Nice job!

**Comment by lemeurherve** [comment]
> don't forget to check the "apply" execution once merged.

All good: 
image

**Pull request merged**